pynicotine/gtkgui/privatechat.py

```diff
@@ -398,20 +399,23 @@ def show_notification(self, text):
         self.chats.highlight_user(self.user)
 
         if config.sections["notifications"]["notification_popup_private_message"]:
+            user_type = _("Buddy") if is_buddy else _("User")
```

Comment:
I would prefer two strings, `Private Message from %(user)s` and `Private Message from Buddy %(user)s`

Reply:
It doesn't seem worth making a new string just for this.

Reply:
Separate strings would be much clearer for a translator. English is very forgiving when hacking together strings like this, but inserting standalone words into another sentence like this wouldn't work in e.g. one of the other languages I know due to declension.